src/open_producten/producttypen/tests/api/test_prijs.py

```diff
@@ -108,6 +108,21 @@ def test_create_prijs_with_prijs_opties(self):
             Decimal("74.99"),
         )
 
+    def test_create_prijs_with_optie_with_id(self):
+        id = uuid.uuid4()
+        data = {
+            "actief_vanaf": datetime.date(2024, 1, 3),
+            "prijsopties": [{"id": id, "bedrag": "74.99", "beschrijving": "spoed"}],
+            "product_type_id": self.product_type.id,
+        }
+
+        response = self.client.post(self.path, data)
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(Prijs.objects.count(), 2)
+        self.assertEqual(PrijsOptie.objects.count(), 1)
+        self.assertNotEqual(PrijsOptie.objects.first().id, id)
```

Comment:
If you expect the count to be 1, you can use `.get` instead
```suggestion
        self.assertNotEqual(PrijsOptie.objects.get().id, id)
```